icu4c/source/i18n/unicode/datefmt.h

```diff
@@ -229,13 +229,13 @@ class U_I18N_API DateFormat : public Format {
      * @return clone, or nullptr if an error occurred
      * @stable ICU 2.0
      */
-    virtual DateFormat* clone() const = 0;
+    //virtual DateFormat* clone() const = 0;
```

Comment:
Same question about this being commented out.

Reply:
virtual DateFormat* clone() const override = 0;